core/src/verify.rs

```diff
@@ -281,6 +281,17 @@ impl CommitSequenceVerifier {
                         agenda.transactions_hash
                     )));
                 }
+                // Verify agenda's last previous_block_hash matches with the real previous block hash
```

Comment:
```suggestion
                // Verify if agenda's last previous_block_hash matches with the real previous block hash to prevent replay attacks
```

Reply:
reflected